packages/front-end/components/Experiment/SinglePage.tsx

```diff
@@ -51,6 +54,39 @@ function getColWidth(v: number) {
   return 3;
 }
 
+function drawMetricRow(
+  m: string,
+  metric: MetricInterface,
+  experiment: ExperimentInterfaceStringDates
+) {
+  const override = applyMetricOverrides(metric, experiment);
+  return (
+    <div key={m} className="ml-2">
+      <span className="mr-1">-</span>
+      <Link href={`/metric/${m}`}>
+        <a className="mr-2 font-weight-bold">{metric?.name}</a>
+      </Link>
+      {metric && (
+        <RightRailSectionGroup
+          type="commaList"
+          title="Conversion Window"
+          style={{ marginLeft: 20, fontSize: 12 }}
+        >
```

Comment:
I don't think we need to use nested `<RightRailSectionGroup>` components. That's going to bring along a lot of unnecessary styles. Probably better to just use a div. Also, prefer using bootstrap classes and html elements instead of inline styles when possible. For this, you could use the class `ml-3` or `<small>` for example.